packages/itwinui-css/src/input-container/input-container.scss

```diff
@@ -157,12 +157,11 @@
 
 /// Icon to be shown near a message.
 @mixin iui-input-icon {
+  @include iui-icons-default;
+  @include iui-icons-color;
   display: flex;
```

Comment:
`iui-icons-display` also might be good here

Reply:
Same as above: `iui-icons-display` uses `display: inline-flex`, not `flex`. This change breaks the test images.